network.c

```diff
@@ -1193,6 +1193,38 @@ static ssize_t network_read_data(struct iio_context_pdata *pdata,
 	return network_recv(io_ctx, dst, len, 0);
 }
 
+static int network_consume(struct iio_context_pdata *pdata)
+{
+	int ret;
+	char dummy[1024];
+
+	set_blocking_mode(pdata->io_ctx.fd, false);
```

Comment:
i know there is some precedence in network.c for this `set_blocking_mode(false) + set_blocking_mode(true)`, but it is a bit weird;

Reply:
I think the main reason for this is to also work with windows. Initially, I was just passing `MSG_DONTWAIT` to `recv` but then I looked for this in windows and it seems to be not supported. `set_blocking_mode`will actually have another implementation on windows...